x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/transactions/transactions_groups_alerts.spec.ts

```diff
@@ -73,7 +73,10 @@ export default function ApiTest({ getService }: DeploymentAgnosticFtrProviderCon
     return response.body as TransactionsGroupsMainStatistics;
   }
 
-  describe('Transaction groups alerts', () => {
+  describe('Transaction groups alerts', function () {
+    // fails on MKI, see https://github.com/elastic/kibana/issues/201531
+    this.tags(['failsOnMKI']);
```

Comment:
We should see if we can fix this instead of skipping it.

Reply:
If you can come up with a short-term fix today, then this would definitely preferable. If it takes longer than that, we'll have to temporarily skip the tests for MKI execution in order to unblock the pipeline which is also used for serverless releases.